### ISSUE TITLE
ATM8 Crash Error - Refined Storage

### ISSUE THREAD

**Issue by candidgamera** [refinedmods/refinedstorage#3602]
### Describe the bug

Yesterday around 1PM, my hosted ATM8 server began a crash loop. One player was logged in and used a teleport commend from FTB Essentials - this spiked the tick time about sixty seconds, according the first crash log - and then when the server came back up, it crashed again. The tick time was still above sixty seconds, but for the second and all subsequent crashes, the crash report points to Refined Storage.

I shut down the server, downloaded a copy of the world, inspected the refined storage dat files with NBT Explorer - one player besides myself has an RS system, and had half a billion redstone stored - I edited the dat file to reduce the stored redstone quantities in the hopes that it was just some mass-production feedback going on, but no luck.

I can get the server not to crash if I edit the server properties file to remove the max tick time limit, but cannot get logged in - connection timeout - even if I do so.

We're running 1.0.23, and I had intended to update us to 1.0.28 today, in fact, but this has obviously ground that effort to a halt.

We're running on Shockbyte, and one confounding factor is that I had the server configured to be a Forge 1.19.2 server with auto-updates, and at some point in the last few weeks, Shockbyte's processes did upgrade the Forge version to 43.3.5, instead of 43.2.14. I don't think that's relevant to the crash, but mention it for completeness.

Crash log visible here : https://mclo.gs/fB3K8i6

Please help me get this back online for myself and my friends. Thank you in advance!

### How can we reproduce this bug or crash?

Since it is a crash on boot, I do not have concrete steps to reproduce the issue. I'm unclear as to what is causing the tick lag specifically beyond what the crash report gives me. 

### What Minecraft version is this happening on?

Minecraft 1.18.2

### What Forge version is this happening on?

43.3.5

### What Refined Storage version is this happening on?

1.11.6

### Relevant log output

```shell
---- Minecraft Crash Report ----
// Uh... Did I do that?

Time: 2023-12-01 04:45:36
Description: Watching Server

java.lang.Error: ServerHangWatchdog detected that a single server tick took 60.00 seconds (should be max 0.05)
	at java.util.HashMap$HashIterator.<init>(HashMap.java:1585) ~[?:?] {}
	at java.util.HashMap$EntryIterator.<init>(HashMap.java:1628) ~[?:?] {}
	at java.util.HashMap$EntrySet.iterator(HashMap.java:1098) ~[?:?] {}
	at java.util.AbstractMap.equals(AbstractMap.java:485) ~[?:?] {re:mixin}
	at java.util.Objects.equals(Objects.java:64) ~[?:?] {re:mixin}
	at net.minecraft.nbt.CompoundTag.equals(CompoundTag.java:433) ~[server-1.19.2-20220805.130853-srg.jar%23803!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:modernfix-common.mixins.json:perf.nbt_memory_usage.CompoundTagMixin,pl:mixin:A}
	at java.util.Objects.equals(Objects.java:64) ~[?:?] {re:mixin}
	at java.util.ArrayList.equalsArrayList(ArrayList.java:561) ~[?:?] {re:mixin}
	at java.util.ArrayList.equals(ArrayList.java:529) ~[?:?] {re:mixin}
	at java.util.Objects.equals(Objects.java:64) ~[?:?] {re:mixin}
	at net.minecraft.nbt.ListTag.equals(ListTag.java:325) ~[server-1.19.2-20220805.130853-srg.jar%23803!/:?] {re:mixin,re:computing_frames,re:classloading}
	at java.util.AbstractMap.equals(AbstractMap.java:492) ~[?:?] {re:mixin}
	at java.util.Objects.equals(Objects.java:64) ~[?:?] {re:mixin}
	at net.minecraft.nbt.CompoundTag.equals(CompoundTag.java:433) ~[server-1.19.2-20220805.130853-srg.jar%23803!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:modernfix-common.mixins.json:perf.nbt_memory_usage.CompoundTagMixin,pl:mixin:A}
	at net.minecraft.world.item.ItemStack.m_41658_(ItemStack.java:435) ~[server-1.19.2-20220805.130853-srg.jar%23803!/:?] {re:mixin,xf:fml:forge:itemstack,re:classloading,xf:fml:forge:itemstack,pl:mixin:APP:kubejs-common.mixins.json:ItemStackMixin,pl:mixin:APP:modernfix-common.mixins.json:bugfix.item_cache_flag.ItemStackMixin,pl:mixin:APP:useitemonblockevent.mixins.json:ItemStackMixin,pl:mixin:APP:apotheosis.mixins.json:ItemStackMixin,pl:mixin:APP:tombstone.mixins.json:ItemStackMixin,pl:mixin:APP:tetra.mixins.json:MixinItemStack,pl:mixin:APP:placebo.mixins.json:ItemStackMixin,pl:mixin:APP:itemfilters-common.mixins.json:ItemStackMixin,pl:mixin:APP:elementalcraft.mixins.json:MixinItemStack,pl:mixin:APP:fastsuite.mixins.json:ItemStackMixin,pl:mixin:APP:quark.mixins.json:ItemStackMixin,pl:mixin:A}
	at com.refinedmods.refinedstorage.apiimpl.util.Comparer.isEqual(Comparer.java:20) ~[refinedstorage-1.11.6.jar%23707!/:1.11.6] {re:classloading}
	at com.refinedmods.refinedstorage.api.util.IComparer.isEqualNoQuantity(IComparer.java:44) ~[refinedstorage-1.11.6.jar%23707!/:1.11.6] {re:classloading}
	at com.refinedmods.refinedstorage.apiimpl.util.ItemStackList.add(ItemStackList.java:41) ~[refinedstorage-1.11.6.jar%23707!/:1.11.6] {re:classloading}
	at com.refinedmods.refinedstorage.apiimpl.util.ItemStackList.add(ItemStackList.java:19) ~[refinedstorage-1.11.6.jar%23707!/:1.11.6] {re:classloading}
	at com.refinedmods.refinedstorage.apiimpl.storage.cache.ItemStorageCache.add(ItemStorageCache.java:76) ~[refinedstorage-1.11.6.jar%23707!/:1.11.6] {re:classloading}
	at com.refinedmods.refinedstorage.apiimpl.storage.cache.ItemStorageCache.invalidate(ItemStorageCache.java:66) ~[refinedstorage-1.11.6.jar%23707!/:1.11.6] {re:classloading}
	at com.refinedmods.refinedstorage.apiimpl.storage.cache.ItemStorageCache.lambda$static$0(ItemStorageCache.java:27) ~[refinedstorage-1.11.6.jar%23707!/:1.11.6] {re:classloading}
	at com.refinedmods.refinedstorage.apiimpl.storage.cache.ItemStorageCache$$Lambda$58828/0x0000000806873b08.accept(Unknown Source) ~[?:?] {}
	at com.refinedmods.refinedstorage.apiimpl.network.NetworkNodeGraph.lambda$invalidate$0(NetworkNodeGraph.java:56) ~[refinedstorage-1.11.6.jar%23707!/:1.11.6] {re:classloading}
	at com.refinedmods.refinedstorage.apiimpl.network.NetworkNodeGraph$$Lambda$58830/0x0000000806874598.accept(Unknown Source) ~[?:?] {}
	at java.lang.Iterable.forEach(Iterable.java:75) ~[?:?] {re:mixin}
	at com.refinedmods.refinedstorage.apiimpl.network.NetworkNodeGraph.invalidate(NetworkNodeGraph.java:56) ~[refinedstorage-1.11.6.jar%23707!/:1.11.6] {re:classloading}
	at com.refinedmods.refinedstorage.apiimpl.network.Network.update(Network.java:210) ~[refinedstorage-1.11.6.jar%23707!/:1.11.6] {re:classloading}
	at com.refinedmods.refinedstorage.apiimpl.network.NetworkListener.onLevelTick(NetworkListener.java:17) ~[refinedstorage-1.11.6.jar%23707!/:1.11.6] {re:classloading}
	at com.refinedmods.refinedstorage.apiimpl.network.__NetworkListener_onLevelTick_LevelTickEvent.invoke(.dynamic) ~[refinedstorage-1.11.6.jar%23707!/:1.11.6] {re:classloading,pl:eventbus:B}
	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:73) ~[eventbus-6.0.3.jar%2351!/:?] {}
	at net.minecraftforge.eventbus.EventBus$$Lambda$4260/0x00000008016eecf8.invoke(Unknown Source) ~[?:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:315) ~[eventbus-6.0.3.jar%2351!/:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:296) ~[eventbus-6.0.3.jar%2351!/:?] {}
	at net.minecraftforge.event.ForgeEventFactory.onPostLevelTick(ForgeEventFactory.java:829) ~[forge-1.19.2-43.3.5-universal.jar%23808!/:?] {re:classloading}
	at net.minecraft.server.MinecraftServer.m_5703_(MinecraftServer.java:872) ~[server-1.19.2-20220805.130853-srg.jar%23803!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:kubejs-common.mixins.json:MinecraftServerMixin,pl:mixin:APP:kubejs-common.mixins.json:inject_resources.MinecraftServerMixin,pl:mixin:APP:modernfix-common.mixins.json:perf.dedicated_reload_executor.MinecraftServerMixin,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin,pl:mixin:APP:structure_gel.mixins.json:MinecraftServerMixin,pl:mixin:APP:ae2.mixins.json:spatial.MinecraftServerMixin,pl:mixin:APP:byg.mixins.json:access.MinecraftServerAccess,pl:mixin:APP:byg.mixins.json:server.MixinMinecraftServer,pl:mixin:APP:elementalcraft.mixins.json:MixinMinecraftServer,pl:mixin:A}
	at net.minecraft.server.dedicated.DedicatedServer.m_5703_(DedicatedServer.java:292) ~[server-1.19.2-20220805.130853-srg.jar%23803!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:nochatreports.mixins.json:server.MixinDedicatedServer,pl:mixin:APP:tombstone.mixins.json:DedicatedServerMixin,pl:mixin:A}
	at net.minecraft.server.MinecraftServer.m_5705_(MinecraftServer.java:806) ~[server-1.19.2-20220805.130853-srg.jar%23803!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:kubejs-common.mixins.json:MinecraftServerMixin,pl:mixin:APP:kubejs-common.mixins.json:inject_resources.MinecraftServerMixin,pl:mixin:APP:modernfix-common.mixins.json:perf.dedicated_reload_executor.MinecraftServerMixin,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin,pl:mixin:APP:structure_gel.mixins.json:MinecraftServerMixin,pl:mixin:APP:ae2.mixins.json:spatial.MinecraftServerMixin,pl:mixin:APP:byg.mixins.json:access.MinecraftServerAccess,pl:mixin:APP:byg.mixins.json:server.MixinMinecraftServer,pl:mixin:APP:elementalcraft.mixins.json:MixinMinecraftServer,pl:mixin:A}
	at net.minecraft.server.MinecraftServer.m_130011_(MinecraftServer.java:654) ~[server-1.19.2-20220805.130853-srg.jar%23803!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:kubejs-common.mixins.json:MinecraftServerMixin,pl:mixin:APP:kubejs-common.mixins.json:inject_resources.MinecraftServerMixin,pl:mixin:APP:modernfix-common.mixins.json:perf.dedicated_reload_executor.MinecraftServerMixin,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin,pl:mixin:APP:structure_gel.mixins.json:MinecraftServerMixin,pl:mixin:APP:ae2.mixins.json:spatial.MinecraftServerMixin,pl:mixin:APP:byg.mixins.json:access.MinecraftServerAccess,pl:mixin:APP:byg.mixins.json:server.MixinMinecraftServer,pl:mixin:APP:elementalcraft.mixins.json:MixinMinecraftServer,pl:mixin:A}
	at net.minecraft.server.MinecraftServer.m_206580_(MinecraftServer.java:244) ~[server-1.19.2-20220805.130853-srg.jar%23803!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:kubejs-common.mixins.json:MinecraftServerMixin,pl:mixin:APP:kubejs-common.mixins.json:inject_resources.MinecraftServerMixin,pl:mixin:APP:modernfix-common.mixins.json:perf.dedicated_reload_executor.MinecraftServerMixin,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin,pl:mixin:APP:structure_gel.mixins.json:MinecraftServerMixin,pl:mixin:APP:ae2.mixins.json:spatial.MinecraftServerMixin,pl:mixin:APP:byg.mixins.json:access.MinecraftServerAccess,pl:mixin:APP:byg.mixins.json:server.MixinMinecraftServer,pl:mixin:APP:elementalcraft.mixins.json:MixinMinecraftServer,pl:mixin:A}
	at net.minecraft.server.MinecraftServer$$Lambda$55345/0x00000008063a1a78.run(Unknown Source) ~[?:?] {}
	at java.lang.Thread.run(Thread.java:833) [?:?] {re:mixin}


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Server Watchdog
Suspected Mod: 
	Refined Storage (refinedstorage), Version: 1.11.6
		Issue tracker URL: https://github.com/refinedmods/refinedstorage
		at TRANSFORMER/refinedstorage@1.11.6/com.refinedmods.refinedstorage.apiimpl.util.Comparer.isEqual(Comparer.java:20)
Stacktrace:
	at java.util.HashMap$HashIterator.<init>(HashMap.java:1585) ~[?:?] {}
	at java.util.HashMap$EntryIterator.<init>(HashMap.java:1628) ~[?:?] {}
	at java.util.HashMap$EntrySet.iterator(HashMap.java:1098) ~[?:?] {}
	at java.util.AbstractMap.equals(AbstractMap.java:485) ~[?:?] {re:mixin}
	at java.util.Objects.equals(Objects.java:64) ~[?:?] {re:mixin}
	at net.minecraft.nbt.CompoundTag.equals(CompoundTag.java:433) ~[server-1.19.2-20220805.130853-srg.jar%23803!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:modernfix-common.mixins.json:perf.nbt_memory_usage.CompoundTagMixin,pl:mixin:A}
	at java.util.Objects.equals(Objects.java:64) ~[?:?] {re:mixin}
	at java.util.ArrayList.equalsArrayList(ArrayList.java:561) ~[?:?] {re:mixin}
	at java.util.ArrayList.equals(ArrayList.java:529) ~[?:?] {re:mixin}
	at java.util.Objects.equals(Objects.java:64) ~[?:?] {re:mixin}
	at net.minecraft.nbt.ListTag.equals(ListTag.java:325) ~[server-1.19.2-20220805.130853-srg.jar%23803!/:?] {re:mixin,re:computing_frames,re:classloading}
	at java.util.AbstractMap.equals(AbstractMap.java:492) ~[?:?] {re:mixin}
	at java.util.Objects.equals(Objects.java:64) ~[?:?] {re:mixin}
	at net.minecraft.nbt.CompoundTag.equals(CompoundTag.java:433) ~[server-1.19.2-20220805.130853-srg.jar%23803!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:modernfix-common.mixins.json:perf.nbt_memory_usage.CompoundTagMixin,pl:mixin:A}
	at net.minecraft.world.item.ItemStack.m_41658_(ItemStack.java:435) ~[server-1.19.2-20220805.130853-srg.jar%23803!/:?] {re:mixin,xf:fml:forge:itemstack,re:classloading,xf:fml:forge:itemstack,pl:mixin:APP:kubejs-common.mixins.json:ItemStackMixin,pl:mixin:APP:modernfix-common.mixins.json:bugfix.item_cache_flag.ItemStackMixin,pl:mixin:APP:useitemonblockevent.mixins.json:ItemStackMixin,pl:mixin:APP:apotheosis.mixins.json:ItemStackMixin,pl:mixin:APP:tombstone.mixins.json:ItemStackMixin,pl:mixin:APP:tetra.mixins.json:MixinItemStack,pl:mixin:APP:placebo.mixins.json:ItemStackMixin,pl:mixin:APP:itemfilters-common.mixins.json:ItemStackMixin,pl:mixin:APP:elementalcraft.mixins.json:MixinItemStack,pl:mixin:APP:fastsuite.mixins.json:ItemStackMixin,pl:mixin:APP:quark.mixins.json:ItemStackMixin,pl:mixin:A}
	at com.refinedmods.refinedstorage.apiimpl.util.Comparer.isEqual(Comparer.java:20) ~[refinedstorage-1.11.6.jar%23707!/:1.11.6] {re:classloading}
	at com.refinedmods.refinedstorage.api.util.IComparer.isEqualNoQuantity(IComparer.java:44) ~[refinedstorage-1.11.6.jar%23707!/:1.11.6] {re:classloading}
	at com.refinedmods.refinedstorage.apiimpl.util.ItemStackList.add(ItemStackList.java:41) ~[refinedstorage-1.11.6.jar%23707!/:1.11.6] {re:classloading}
	at com.refinedmods.refinedstorage.apiimpl.util.ItemStackList.add(ItemStackList.java:19) ~[refinedstorage-1.11.6.jar%23707!/:1.11.6] {re:classloading}
	at com.refinedmods.refinedstorage.apiimpl.storage.cache.ItemStorageCache.add(ItemStorageCache.java:76) ~[refinedstorage-1.11.6.jar%23707!/:1.11.6] {re:classloading}
	at com.refinedmods.refinedstorage.apiimpl.storage.cache.ItemStorageCache.invalidate(ItemStorageCache.java:66) ~[refinedstorage-1.11.6.jar%23707!/:1.11.6] {re:classloading}
	at com.refinedmods.refinedstorage.apiimpl.storage.cache.ItemStorageCache.lambda$static$0(ItemStorageCache.java:27) ~[refinedstorage-1.11.6.jar%23707!/:1.11.6] {re:classloading}
	at com.refinedmods.refinedstorage.apiimpl.storage.cache.ItemStorageCache$$Lambda$58828/0x0000000806873b08.accept(Unknown Source) ~[?:?] {}
	at com.refinedmods.refinedstorage.apiimpl.network.NetworkNodeGraph.lambda$invalidate$0(NetworkNodeGraph.java:56) ~[refinedstorage-1.11.6.jar%23707!/:1.11.6] {re:classloading}
	at com.refinedmods.refinedstorage.apiimpl.network.NetworkNodeGraph$$Lambda$58830/0x0000000806874598.accept(Unknown Source) ~[?:?] {}
	at java.lang.Iterable.forEach(Iterable.java:75) ~[?:?] {re:mixin}
	at com.refinedmods.refinedstorage.apiimpl.network.NetworkNodeGraph.invalidate(NetworkNodeGraph.java:56) ~[refinedstorage-1.11.6.jar%23707!/:1.11.6] {re:classloading}
	at com.refinedmods.refinedstorage.apiimpl.network.Network.update(Network.java:210) ~[refinedstorage-1.11.6.jar%23707!/:1.11.6] {re:classloading}
	at com.refinedmods.refinedstorage.apiimpl.network.NetworkListener.onLevelTick(NetworkListener.java:17) ~[refinedstorage-1.11.6.jar%23707!/:1.11.6] {re:classloading}
	at com.refinedmods.refinedstorage.apiimpl.network.__NetworkListener_onLevelTick_LevelTickEvent.invoke(.dynamic) ~[refinedstorage-1.11.6.jar%23707!/:1.11.6] {re:classloading,pl:eventbus:B}
	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:73) ~[eventbus-6.0.3.jar%2351!/:?] {}
	at net.minecraftforge.eventbus.EventBus$$Lambda$4260/0x00000008016eecf8.invoke(Unknown Source) ~[?:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:315) ~[eventbus-6.0.3.jar%2351!/:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:296) ~[eventbus-6.0.3.jar%2351!/:?] {}
	at net.minecraftforge.event.ForgeEventFactory.onPostLevelTick(ForgeEventFactory.java:829) ~[forge-1.19.2-43.3.5-universal.jar%23808!/:?] {re:classloading}
	at net.minecraft.server.MinecraftServer.m_5703_(MinecraftServer.java:872) ~[server-1.19.2-20220805.130853-srg.jar%23803!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:kubejs-common.mixins.json:MinecraftServerMixin,pl:mixin:APP:kubejs-common.mixins.json:inject_resources.MinecraftServerMixin,pl:mixin:APP:modernfix-common.mixins.json:perf.dedicated_reload_executor.MinecraftServerMixin,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin,pl:mixin:APP:structure_gel.mixins.json:MinecraftServerMixin,pl:mixin:APP:ae2.mixins.json:spatial.MinecraftServerMixin,pl:mixin:APP:byg.mixins.json:access.MinecraftServerAccess,pl:mixin:APP:byg.mixins.json:server.MixinMinecraftServer,pl:mixin:APP:elementalcraft.mixins.json:MixinMinecraftServer,pl:mixin:A}
	at net.minecraft.server.dedicated.DedicatedServer.m_5703_(DedicatedServer.java:292) ~[server-1.19.2-20220805.130853-srg.jar%23803!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:nochatreports.mixins.json:server.MixinDedicatedServer,pl:mixin:APP:tombstone.mixins.json:DedicatedServerMixin,pl:mixin:A}
	at net.minecraft.server.MinecraftServer.m_5705_(MinecraftServer.java:806) ~[server-1.19.2-20220805.130853-srg.jar%23803!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:kubejs-common.mixins.json:MinecraftServerMixin,pl:mixin:APP:kubejs-common.mixins.json:inject_resources.MinecraftServerMixin,pl:mixin:APP:modernfix-common.mixins.json:perf.dedicated_reload_executor.MinecraftServerMixin,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin,pl:mixin:APP:structure_gel.mixins.json:MinecraftServerMixin,pl:mixin:APP:ae2.mixins.json:spatial.MinecraftServerMixin,pl:mixin:APP:byg.mixins.json:access.MinecraftServerAccess,pl:mixin:APP:byg.mixins.json:server.MixinMinecraftServer,pl:mixin:APP:elementalcraft.mixins.json:MixinMinecraftServer,pl:mixin:A}
	at net.minecraft.server.MinecraftServer.m_130011_(MinecraftServer.java:654) ~[server-1.19.2-20220805.130853-srg.jar%23803!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:kubejs-common.mixins.json:MinecraftServerMixin,pl:mixin:APP:kubejs-common.mixins.json:inject_resources.MinecraftServerMixin,pl:mixin:APP:modernfix-common.mixins.json:perf.dedicated_reload_executor.MinecraftServerMixin,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin,pl:mixin:APP:structure_gel.mixins.json:MinecraftServerMixin,pl:mixin:APP:ae2.mixins.json:spatial.MinecraftServerMixin,pl:mixin:APP:byg.mixins.json:access.MinecraftServerAccess,pl:mixin:APP:byg.mixins.json:server.MixinMinecraftServer,pl:mixin:APP:elementalcraft.mixins.json:MixinMinecraftServer,pl:mixin:A}
	at net.minecraft.server.MinecraftServer.m_206580_(MinecraftServer.java:244) ~[server-1.19.2-20220805.130853-srg.jar%23803!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:kubejs-common.mixins.json:MinecraftServerMixin,pl:mixin:APP:kubejs-common.mixins.json:inject_resources.MinecraftServerMixin,pl:mixin:APP:modernfix-common.mixins.json:perf.dedicated_reload_executor.MinecraftServerMixin,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin,pl:mixin:APP:structure_gel.mixins.json:MinecraftServerMixin,pl:mixin:APP:ae2.mixins.json:spatial.MinecraftServerMixin,pl:mixin:APP:byg.mixins.json:access.MinecraftServerAccess,pl:mixin:APP:byg.mixins.json:server.MixinMinecraftServer,pl:mixin:APP:elementalcraft.mixins.json:MixinMinecraftServer,pl:mixin:A}
```

**Comment by candidgamera** [comment]
Update : I manually edited the refined storage network dat file to put both RS networks in redstone-control mode. This allowed the server to load with both networks turned off and I was able to log in.

**Comment by codegavran** [comment]
Getting the same error on a standard ATM9 server running RS 1.12.4

Once you were able to get in, what did you do to prevent the crash from reoccurring @candidgamera ?

**Comment by candidgamera** [comment]
> Getting the same error on a standard ATM9 server running RS 1.12.4
> 
> Once you were able to get in, what did you do to prevent the crash from reoccurring @candidgamera ?

@codegavran  - Having had inspected the various RS dat files, I knew that it was unlikely to be my own network causing the crash - I just didn't have anything heinous in storage. I took the precaution of removing any creative exporters I had on my network, to guarantee I didn't have any creative exporter/importer race conditions going on, and re-enabled mine. Once that happened safely, I turned my attention to the other RS network on the server - my friend had been storing a ton of equipment drops from his mob farm onto an infinite drive in his RS system. I had him remove the drive and make sure nothing else was being imported before we re-enabled his. 

I still don't have a concrete sense of what /exactly/ the issue was, but removing the thousands of equipment objects with NBT data from his system seems to have kept the issue from recurring for us.
